binder/apt.txt

```diff
@@ -1,2 +1,2 @@
-openjdk-8-jre
+openjdk-17-jre
```

Comment:
Hm, this was actually fixed by https://github.com/apache/spark/commit/bb0d287114fbebf0c5457cee066c5080735a48a4 :-).